modules/azure/function_app_linux_managed_identity/variables.tf

```diff
@@ -76,6 +76,13 @@ variable "storage_account_name" {
 variable "storage_account_access_key" {
   type        = string
   description = " The access key which will be used to access the backend storage account for the Function App."
+  default     = null
+}
+
+variable "use_managed_identity" {
+  type        = bool
+  description = "Adds managed identity to the function app, that allows to avoid explicit access token usage"
+  default     = true
```

Comment:
Yeah, you are technically rights it is a change how the module works. However I would argue it is not really a breaking change. It just changes "how", under the hood, you provide the access to storage blob containing zip . You really want to use Managed identity as a best practice, instead of access Key. 

So even if I keep it true, the module users  behavior, getting blob with source code, would still retrieve this blob successfully, meaning it is not breaking.